appside/data/sample_app.js

```diff
@@ -92,6 +92,17 @@ var SAMPLE_APP = {
         }
     },
 
+
+    'likert': {
+        'title': '',       // should come from constants.js
+        'instructions': '',    // should come from constants.js
+
+        'questions': [          // provided in constructor
+            ['Question 1', undefined],    // question text, user response 1-5
```

Comment:
suggest setting one of these to a value not undefined. Frame should read the value and select the appropriate radio button

use case: user selects some answers, navigates to next page, then navigates back. Same answers should be shown on the page that they navigate back to.

Reply:
changed.